gcp/modules/vpc/variables.tf

```diff
@@ -0,0 +1,69 @@
+variable "name" {
```

Comment:
We need to have permittable instances, please.  Set a variable to turn this on or off.  E2E will use cheaper instances.

Reply:
File an issue to have https://cloud.google.com/compute/docs/instances/preemptible instances